### PR TITLE
apply indend fix, change location of board name define

### PR DESCRIPTION
**Pull request by trivalik**
Here the recreate of previous pull request.

### CODE REVIEW

File: Marlin/pins_ANET_10.h [hunk 1]
```diff
@@ -24,19 +24,17 @@
...
 #include "pins_SANGUINOLOLU_12.h"
 
-#undef BOARD_NAME
-#define BOARD_NAME "Anet"
-
 #undef LED_PIN
 #undef FAN_PIN
 
-#define LED_PIN -1
 #define FAN_PIN 4
```
Comment (oderwat) [inline Marlin/pins_ANET_10.h]: without LED_PIN defined you lose SD Card functionality. As I said before, I think we need an pins file which is not dependent on other files.

File: Marlin/ultralcd.cpp [hunk 2]
```diff
@@ -3504,59 +3504,64 @@ void lcd_update() {
...
+        if (buttons_reprapworld_keypad != 0) {
+          adc_steps++;
+          if(adc_steps > 20) {
+            adc_steps = 20;
+          }
+          lcd_quick_feedback();
+          lcdDrawUpdate = LCDVIEW_REDRAW_NOW;
+          return_to_status_ms = millis() + LCD_TIMEOUT_TO_STATUS;
+          if (encoderDirection == -1) { // side effect which signals we are inside a menu
+            if (buttons_reprapworld_keypad & EN_REPRAPWORLD_KEYPAD_DOWN) {
```
Comment (oderwat) [inline Marlin/ultralcd.cpp:3518]: I said it before. I don't want additional {} here. These are not matching the current style of the file.

File: Marlin/pins_ANET_10.h (hunk 1 above)
Comment (trivalik) [reply Marlin/pins_ANET_10.h]: thinkyhead was reviewing my code and told it. So just create a pull request, and you will see what the say.

File: Marlin/ultralcd.cpp (hunk 2 above)
Comment (trivalik) [reply Marlin/ultralcd.cpp:3518]: It's up to you, to using brackets is always recommend. thinkyhead will add it anyway later.

**Comment by oderwat** [comment]
I close this as there is now a PR in Marlin.

**Comment by G-Pereira** [comment]
Can you provide a link to the specific PR? I can't find it

**Comment by benlye** [comment]
https://github.com/MarlinFirmware/Marlin/pull/7016

**Comment by G-Pereira** [comment]
hm thanks, why the name Anet A10?

**Comment by benlye** [comment]
It was a bastardization of the Anet v1.0 board name in the Marlin config files - ANET_10.  There is no A10 ;-)

**Comment by G-Pereira** [comment]
Oh, of course! :smile:

**Pull request closed without merging**